prisma/migrations/20220707113138_is_editted_corrected_to_is_edited/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `isEditted` on the `Note` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "Note" DROP COLUMN "isEditted",
```

Comment:
This is problematic - how are you going to handle the data loss? Currently the production table looks like this:
![Screenshot 2022-07-07 at 12 41 16](https://user-images.githubusercontent.com/13719921/177765199-156b7de6-521c-44f9-ba06-2f95344e6715.png)

Ha, ok - so because noone has used it yet, you're in luck. No change needed. Normally this would be lossy, and we'd have to

1. Add the new column (no dropping yet)
2. Copy all data over to the new column
3. update any existing code to reflect the new property name
4. Then drop the column

Reply:
It was a spelling mistake in the schema (luckily caught early by Carlo), there is no way at the moment to add a note on the live app as the front end has not been started

Reply:
Careful - it's an API - you can interact with it with any client, not just the front end app we're building. It's entirely possible there could be data added to this table. (It just so happens that noone is making requests to the prod API from another client (curl, insomnia etc)